CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to the Deploy to Azure extension will be documented in this
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/). Versioning follows an internal Azure DevOps format that is not compatible with SemVer.
 
+## 1.0.7
+### Added
+- Integrated with repo analysis and template service
```

Comment:
should we be more verbose here? YOu can say that not the extension does better analysis of the repo and based on that analysis it suggests a weighted list of more accurate templates. It uses backend services for both repo analysis and template management

Reply:
Have updated the text by cross-checking with Pulkit